### PR TITLE
Adding support for SPM

### PR DESCRIPTION
**Pull request by DominatorVbN**
This merge request add support for Swift Package Manager, 
As many of iOS Developers have already switched to SPM for managing dependencies, 
and as I am also developing a side project whose ui has multiple Neomorphic UI Elements and I am using Synth for it, as my project is using SPM, thus adding support for SPM in Synth.

### CODE REVIEW

**Review by Khemist-sis — APPROVED** [review]

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Khemist-sis) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: Encoded

Comment (Khemist-sis) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: Yes

**Review by Khemist-sis — APPROVED** [review]

**Comment by DominatorVbN** [comment]
Hey, Guys
Is there anything wrong with the PR ?

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (Enxhi369) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: Yea

**Review by Khemist-sis — APPROVED** [review]

**Review by Khemist-sis — APPROVED** [review]

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "Synth",
```
Comment (prashant-shrivastava91) [inline Package.swift:6]: can you rename all instances of library name from `Synth` to `synth-ios`

**Review by prashant-shrivastava91 — CHANGES_REQUESTED** [review]

File: Package.swift (hunk 2 above)
Comment (DominatorVbN) [reply Package.swift:6]: Renamed all instances to synth-ios in commit c7fc339

**Comment by DominatorVbN** [comment]
I have reverted the directory structure while renaming the framework's target and binary name to 'synth-ios'

**Review by prashant-shrivastava91 — APPROVED** [review]

**Pull request merged**